### ISSUE TITLE
Change LOW severity color to blue

### ISSUE THREAD

**Issue by 3nvi** [panther-labs/panther#1396]
### Description

Because of lack of proper color contrast, we will be changing the color of our badges for "low" severity to blue

### Designs

https://app.abstract.com/projects/558a8de6-7134-4c8b-91c5-62074bb1279b/branches/849670d3-62c0-462f-b399-bf6174c4352b/commits/latest/files/c2bbc1a8-4778-4cd6-93a5-4630d8868201/layers/C33133F6-EFC3-41AF-9175-5DF64C2EBD72?collectionId=eb762129-4a3e-4947-85cb-4905f141cb33&collectionLayerId=db74ad66-9991-4ee8-af93-d216a34a14f0

### Acceptance Criteria

- LOW severity  is always depicted using a blue color
